docs/api-docs/headless/end-to-end-guide/graphql.mdx

```diff
@@ -272,17 +272,17 @@ You can access Cart and Checkout details in the same request as other informatio
     Authorization: Bearer {{CUSTOMER_IMPERSONATION_TOKEN}}
     X-Bc-Customer-Id: 123
 
-    query getCheckout {
+    query getCheckout($entityId: Int) {
       site {
-        checkout {
+        checkout(entityId: $entityId) {
           entityId
           billingAddress {
             ...CheckoutBillingAddressFields
           }
           shippingConsignments {
             ...CheckoutShippingConsignmentFields
           }
-          order {
+          order(entityId: $entityId) {
```

Comment:
This is incorrect.

Passing in an entity ID as an argument is not the same as querying an entity ID.  That's two different concepts, so there's not a need for adding an argument everywhere that entityId is being queried.  

```
order {
  entityId
}
```

This is saying "give me the entityId of the order".  No argument involved. So it should remain as it was.